shell/browser/feature_list.cc

```diff
@@ -25,6 +25,11 @@ void InitializeFeatureList() {
   // when node integration is enabled.
   disable_features +=
       std::string(",") + features::kSpareRendererForSitePerProcess.name;
+
+#if defined(OS_WIN)
+  disable_features +=
+      std::string(",") + features::kLegacyWindowsDWriteFontFallback.name;
```

Comment:
Why are we disabling this feature ?

Reply:
https://github.com/electron/electron/pull/19792#discussion_r316027352

Reply:
That comment doesn't explain why we are disabling it.

Reply:
I disabled it to fix the crash, since i don't believe we support that fallpath path in Electron. I felt it would be easier to remove it since it was previously disabled by default.

Reply:
@codebytere disabling the feature doesn't restore the old behavior, after https://chromium-review.googlesource.com/c/chromium/src/+/1708138

```
Organise Blink font fallback into two main functions:

  * Retrieving a fallback font from the legacy path which finds a
    fallback font from hard-coded lists that have been curated manually
    and maintained.

  * Retrieving a fallback font through DWrite fallback API, either by
    calling Skia API in-process, which in turn uses DWriteFontProxy out
    of process bridging, or calling a Mojo method to retrieve a fallback
    family name.

Use the LegacyWindowsDWriteFontFallbackEnabled RunTimeEnabledFeature to
decide which fallback font to return, if the flag is on, always use
DWrite. If the flag is off, on Windows 8 and below, only use the
hard-coded fallback path. On Windows 8.1 and above, prioritze the
hard-coded path, but try the API path if the hard-coded path did not
find a result. This is intended as an interim flag to control which
fallback font code path to use and to measure quality and availability
of fallback fonts in the new code path. The flag is to be removed once
we are confident to switch to the new code.
```

So for windows 8.1 and above it will try the new api path irrespective of the flag, as the flag only has effect for older windows version.

I would suggest to fix the crash and enable this flag. As chromium seems to have enabled the api code path for all windows versions https://chromium-review.googlesource.com/c/chromium/src/+/1753006.

Reply:
https://github.com/electron/electron/pull/19792/commits/3b5bd192b464cfc30642500826ec1ec20511880f i think this should do it @deepak1556